src/server.c

```diff
@@ -5584,9 +5585,11 @@ sds genValkeyInfoStringACLStats(sds info) {
                         "acl_access_denied_auth:%lld\r\n"
                         "acl_access_denied_cmd:%lld\r\n"
                         "acl_access_denied_key:%lld\r\n"
-                        "acl_access_denied_channel:%lld\r\n",
+                        "acl_access_denied_channel:%lld\r\n"
+                        "invalid_tls_cert_auth_failures:%lld\r\n",
```

Comment:
Oh, this is a new INFO field. This is also part of the user-facing API, so we should be careful.

This is not actually counting invalid certificates. Invalid certificate means that it is rejected, for example if it's expired or not trusted by the CA certs. In that case, it would be rejected in the TLS handshake. Here it can be valid but just not match an existing username.

Probably it should start with "acl" like the other fields here.

How about `"acl_access_denied_tls_cert"`? @madolson please chime in.

Reply:
@zuiderkwast @madolson 
But this is not like any ACL rule which has caused access denied so like having `acl_access_denied` would be misleading, may be, `auto_tls_cert_auth_failures` ?

Reply:
In a way it is just like a failed AUTH command. When the CommonName doesn't match an existing user, you get connected as the default user, which is the same as connecting without AUTH as well as after a failed AUTH command. If ACL is used at all, then typically the default user doesn't have access to anything other than trying to call AUTH again (and a few more commands like HELLO).

Reply:
@zuiderkwast @madolson can we finalize something on this ?

Reply:
Great, seems we can agree on this. Consider it decided.